packages/hub/node-tests/routes/email-card-drop-verify-test.ts

```diff
@@ -96,13 +98,39 @@ describe('GET /email-card-drop/verify', function () {
     expect(jobPayloads).to.be.empty;
   });
 
+  it('rejects an unknown email-hash', async function () {
+    let response = await request().get(
+      `/email-card-drop/verify?eoa=${unclaimedEoa.ownerAddress}&verification-code=${unclaimedEoa.verificationCode}&email-hash=wha`
+    );
+
+    expect(response.status).to.equal(400);
+    expect(response.text).to.equal('Email is invalid');
```

Comment:
If we are not redirecting, then our error messages might be confusing to a user. We could go with something like a generic "Card drop link is invalid".

Reply:
It’s easy to change these, I’ll leave it for future possible improvement